spec/index.bs

```diff
@@ -2656,6 +2656,8 @@ has a default layout created and used instead.
 
                 1. Set |entry|.{{GPUBindGroupLayoutEntry/hasDynamicOffset}} to false.
 
+                1. Set |entry|.{{GPUBindGroupLayoutEntry/minBufferBindingSize}} to |resource|'s minimum buffer binding size.
```

Comment:
Would be ideal to explain this in some more detail, but maybe it shouldn't be here and now.

Just add an issue?
```suggestion
                1. Set |entry|.{{GPUBindGroupLayoutEntry/minBufferBindingSize}} to |resource|'s minimum buffer binding size.

                    Issue: link to a definition for "minimum buffer binding size" in the "reflection information".
```